app.py

```diff
@@ -223,6 +223,17 @@ def populate_shotgun(self, session_id, info):
         
         # done!
         self._submission_done = True
+
+        # Log usage metrics
+        try:
+            self.log_metric("Version Create")
```

Comment:
Not sure, but thinking maybe "Flame Sequence Export" is more illustrative? Or is that just repeating the app name?